### PR TITLE
Slider component

### PR DESCRIPTION
**Pull request by Tanay-J**
This PR adds Slider component to ink UI component library
Types of Sliders
- Stepped Slider
- Smooth Slider

**Preview**
image

### CODE REVIEW

**Review by k-rahul21 — COMMENTED** [review]
Small PR. Not much to review. Ready to merge.

**Review by abhij1607 — COMMENTED** [review]
Very neat & clean code || all good practices followed

File: components/slider.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.slider-container {
+  width: 100%;
```
Comment (MonikaJhingan) [inline components/slider.css:2]: In place of % you can use rem for bigger widths and heights but for smaller units percentage is good enough . The code quality looks good and it is very well indented. Just  make the percentages in rems.

**Review by harshalini — APPROVED** [review]
Clean code, merge it

**Review by MonikaJhingan — APPROVED** [review]

File: components/slider.css (hunk 1 above)
Comment (Tanay-J) [reply components/slider.css:2]: Used % as I wanted the element to cover complete width of the parent

**Pull request merged**